fluent/handler.py

```diff
@@ -93,16 +93,23 @@ def __init__(self,
                  host='localhost',
                  port=24224,
                  timeout=3.0,
-                 verbose=False):
+                 verbose=False,
+                 parse_json=False):
 
         self.tag = tag
         self.sender = sender.FluentSender(tag,
                                           host=host, port=port,
                                           timeout=timeout, verbose=verbose)
+        self.parse_json = parse_json
         logging.Handler.__init__(self)
 
     def emit(self, record):
         data = self.format(record)
+        if self.parse_json and isinstance(data, basestring):
```

Comment:
`basestring` was added in 2.3 according to https://docs.python.org/2/library/functions.html#basestring

Haven't tested this PR but would love to push support for json structures forward. Can I help somehow?